src/uint/pow_mod.rs

```diff
@@ -0,0 +1,42 @@
+// use crate::{Concat, Split, UInt};
```

Comment:
Perhaps remove this file and PR it separately?

Reply:
Oops, my bad. This is old code that was not supposed to be pushed. 😅